spec/elements/reinforcement/straight_longitudinal_layer_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+require 'elements/reinforcement/straight_longitudinal_layer'
+require 'engineering/locations/relative'
+require 'elements/reinforcement/rebar'
+require 'materials/steel'
+
+RSpec.describe Elements::Reinforcement::StraightLongitudinalLayer do
+  xit 'does something'
```

Comment:
specs missing?

Reply:
in progress

Reply:
Please take a look; I was waiting to complete some conversation about the class, and now I added the specs